### PR TITLE
Get rid of sle15_workarounds test module

### PR DESCRIPTION
**Pull request by jknphy**
- Related ticket: [poo#75328](https://progress.opensuse.org/issues/75328)
- Verification run: [multiple groups](https://openqa.suse.de/tests/overview?build=jknphy%2Fos-autoinst-distri-opensuse%23sle15_workarounds&distri=sle)

### CODE REVIEW

File: schedule/yast/nvme.yaml [hunk 1]
```diff
@@ -26,7 +26,6 @@ schedule:
   - installation/reboot_after_installation
   - installation/grub_test
   - installation/first_boot
-  - console/sle15_workarounds
```
Comment (rwx788) [inline schedule/yast/nvme.yaml:29]: `system_prepare` module is not scheduled here, we should add it.

File: schedule/qam/15/qam-regression-installation.yaml [hunk 2]
```diff
@@ -25,7 +25,6 @@ schedule:
 - installation/first_boot
 - x11/x11_setup
 - qa_automation/patch_and_reboot
-- console/sle15_workarounds
```
Comment (rwx788) [inline schedule/qam/15/qam-regression-installation.yaml:28]: `system_prepare` module is not scheduled here, we should add it.

File: schedule/qam/15/qam-create_hdd_ha_textmode.yaml [hunk 3]
```diff
@@ -27,7 +27,6 @@ schedule:
   - '{{system_prepare}}'
   - '{{patch_and_reboot_inci}}'
   - '{{patch_and_reboot_test_repo}}'
-  - console/sle15_workarounds
```
Comment (rwx788) [inline schedule/qam/15/qam-create_hdd_ha_textmode.yaml:30]: `system_prepare` module is scheduled conditionally here.

File: schedule/qam/15-SP2/qam-regression-installation.yaml [hunk 4]
```diff
@@ -25,7 +25,6 @@ schedule:
 - installation/first_boot
 - x11/x11_setup
 - qa_automation/patch_and_reboot
-- console/sle15_workarounds
```
Comment (rwx788) [inline schedule/qam/15-SP2/qam-regression-installation.yaml:28]: `system_prepare` module is not scheduled here, we should add it.

File: schedule/qam/15-SP2/qam-create_hdd_ha_textmode.yaml [hunk 5]
```diff
@@ -27,7 +27,6 @@ schedule:
   - '{{system_prepare}}'
   - '{{patch_and_reboot_inci}}'
   - '{{patch_and_reboot_test_repo}}'
-  - console/sle15_workarounds
```
Comment (rwx788) [inline schedule/qam/15-SP2/qam-create_hdd_ha_textmode.yaml:30]: `system_prepare` module is scheduled conditionally here.

File: schedule/qam/15-SP1/qam-regression-installation.yaml [hunk 6]
```diff
@@ -25,7 +25,6 @@ schedule:
 - installation/first_boot
 - x11/x11_setup
 - qa_automation/patch_and_reboot
-- console/sle15_workarounds
```
Comment (rwx788) [inline schedule/qam/15-SP1/qam-regression-installation.yaml:28]: `system_prepare` module is not scheduled here, we should add it.

File: schedule/qam/15-SP1/qam-create_hdd_ha_textmode.yaml [hunk 7]
```diff
@@ -27,7 +27,6 @@ schedule:
   - '{{system_prepare}}'
   - '{{patch_and_reboot_inci}}'
   - '{{patch_and_reboot_test_repo}}'
-  - console/sle15_workarounds
```
Comment (rwx788) [inline schedule/qam/15-SP1/qam-create_hdd_ha_textmode.yaml:30]: `system_prepare` module is scheduled only conditionally here.

File: lib/main_common.pm [hunk 8]
```diff
@@ -1926,8 +1926,6 @@ sub load_x11_installation {
     load_reboot_tests();
     loadtest "x11/x11_setup";
     loadtest 'qa_automation/patch_and_reboot' if is_updates_tests;
-    # temporary adding test modules which applies hacks for missing parts in sle15
-    loadtest "console/sle15_workarounds" if is_sle('15+');
```
Comment (rwx788) [inline lib/main_common.pm:1930]: I'm not sure if we should just exclude it, but rather replace with `system_prepare`.

**Review by foursixnine — APPROVED** [review]
Thanks for taking care and verifying in multiple job groups :)

**Review by rwx788 — APPROVED** [review]
LGTM

**Pull request merged**